cmd/argo/commands/retry.go

```diff
@@ -58,7 +60,23 @@ func NewRetryCommand() *cobra.Command {
 				log.Fatalf("Unable to parse node field selector '%s': %s", retryOps.nodeFieldSelector, err)
 			}
 
-			for _, name := range args {
+			var names []string
+
+			if retryOps.labelSelector != "" || retryOps.fieldSelector != "" {
+				listed, err := listWorkflows(ctx, serviceClient, listFlags{
```

Comment:
like how you re-use a pattern here

Reply:
@alexec Do you meant to use the `listFlags` struct as the [delete.go](https://github.com/argoproj/argo/blob/master/cmd/argo/commands/delete.go#L54) file? If not, do you recommend the way how to re-use it? :)

Reply:
I think submit new PRs as you see fit for each change. Don't worry about re-use just yet.